jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxterrgv5.l

```diff
@@ -0,0 +1,205 @@
+;; vim: set ft=lisp:
+
+(require "package://baxtereus/baxter-util.l")
+(require "package://jsk_2016_01_baxter_apc/robots/baxterrgv5.l")
+
+(unless (find-package "JSK_2016_01_BAXTER_APC")
+  (make-package "JSK_2016_01_BAXTER_APC"))
+
+(defclass jsk_2016_01_baxter_apc::baxter-robot
```

Comment:
> ? I am just saying you can name the class jsk_2016_01_baxter_apc::baxterrgv5.

Ok. I want to unite names, so I should change:
* Add `baxterrgv5-robot` class for `jsk_2016_01_baxter_apc::baxterrgv5-robot`'s parent which has `baxterrgv5` robot model and utils in `baxter-util.l`.
* jsk_2016_01_baxter_apc::baxter-interface -> jsk_2016_01_baxter_apc::baxterrgv5-interface
* `*baxter*` -> `*baxterrgv5*`

> BTW, what is rgv5 in baxterrgv5? right gripper version 5?

Yes, exactly.